packages/bpk-docs/src/pages/ModalsPage/ModalsPage.js

```diff
@@ -121,7 +121,218 @@ const components = [
         buttonText="Open modal"
       >
         <Paragraph>
-          You can put anything you want in here, including forms:
```

Comment:
Hmm I'm not sure about having this as an example in the docs - perhaps a story is enough?